cli/src/semgrep/app/scans.py

```diff
@@ -213,6 +215,15 @@ def report_findings(
                 "errors": [error.to_dict() for error in errors],
                 "total_time": total_time,
                 "unsupported_exts": dict(ignored_ext_freqs),
+                "parse_rate": {
+                    lang: {
+                        "targets_parsed": data.num_targets - data.targets_with_errors,
```

Comment:
Nit: `targets_parsed` could end up negative which could mess up data. 
```suggestion
                        "targets_parsed": min(data.num_targets - data.targets_with_errors,0),
```

Reply:
`targets_with_errors` is necessarily less by construction, so I don't think adding an extra check here to hide a failure is the right choice.